normandy/settings.py

```diff
@@ -195,6 +194,9 @@ class Base(Core, CORS, OIDC):
             # We've subclassed Django's security middleware, so Django's
             # checks can't tell we are using the middleware.
             "security.W001",
+            # We have disabled CSRF as we do not use cookie-based authentication.
+            # As a result the CSRF middleware check is not required.
+            "security.W003",
             # Check CSRF cookie http only. disabled because we read the
             # CSRF cookie in JS for forms in React.
             "security.W017",
```

Comment:
Can we stop overriding the HTTP only checks, and then remove W017 from this list?